### ISSUE TITLE
Image causes text to be off-center

### ISSUE THREAD

**Issue by ThomasSunderland** [scottyab/showhidepasswordedittext#6]
If you have two text fields one on top of the other with one being a regular EditText field and the other of this custom type and both have a gravity of center; then the text will not line up once the show/hide image appears.

**Comment by scottyab** [comment]
Hey @ThomasSunderland thanks for raising this, do you have code/layout and/or screen shot of this issue.

**Comment by ThomasSunderland** [comment]
Yes, it's a little hard to see in this one, but you can see that the text in the ShowHidePasswordEditText field is not centered with the text in the EditText field above it despite both of the fields being in line with one another and both having a gravity of center.

![loginscreen](https://cloud.githubusercontent.com/assets/2091370/12485165/8f693c2a-c02b-11e5-8998-6ff613602a60.png)

**Comment by scottyab** [comment]
I guess a quick fix would be to add padding to the left of passsword edit text?

**Comment by ThomasSunderland** [comment]
I think I had tried that and it moved the hide/show icon over as well, but I could be wrong
